app/views/case_study/show.html.erb

```diff
@@ -34,6 +34,10 @@
                 <%= render 'layouts/components/full_bleed_image', alt_text: 'example image', image_src: content[:data][:url] %>
               </div>
             <% end %>
+          <% when :testimonial %>
+            <div class="govuk-grid-row padding-top-m  width-s">
+              <%= render 'layouts/components/testimonial', testimonial: content[:data][:testimonial] %>
```

Comment:
`content[:data][:testimonial]` should be `content[:data]` to line up with what the case-study use case will return.